plugins/lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "packages": ["*/"],
```

Comment:
Before this was "**" which failed because `postinstall` in the package.json got into an infinite loop. Now it properly only treats subdirectories as packages =)